scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
+              help='Portal to add the analyses to')
+@click.option('--analyses', required=True, type=str,
```

Comment:
Got it, did not know that. Maybe both options should become arguments?
... 

On (Jul-01-15| 9:12), Joshua Shorenstein wrote:
> > @@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
> >                                              preprocessed_data)
> >      click.echo("Preprocessed data %s successfully updated" % ppd.id)
> >  
> > +# #############################################################################
> > +# PORTAL COMMANDS
> > +# #############################################################################
> > +
> > +
> > +@portal.command(name="add-analyses")
> > +@click.option('--portal', required=True, type=str,
> > +              help='Portal to add the analyses to')
> > +@click.option('--analyses', required=True, type=str,
> 
> nargs only works for arguments, not options.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1316/files#r33694871